metadata/scale.xml

```diff
@@ -39,5 +39,10 @@
 			<min>0.0</min>
 			<max>1.0</max>
 		</option>
+		<option name="middle_click_close" type="bool">
+			<_short>Close windows with middle click</_short>
+			<_long>Use the middle mouse button to close windows in the scale view.</_long>
```

Comment:
The terminology in wayfire is *views* instead of *windows*.

Reply:
Ok, I'm changing it; I wasn't sure if this was used in user-facing parts as well